pkg/yurtctl/util/edgenode/common.go

```diff
@@ -60,29 +60,34 @@ metadata:
   namespace: kube-system
 spec:
   volumes:
-  - name: hub-dir
+  - name: pki
     hostPath:
-      path: /var/lib/yurthub
-      type: DirectoryOrCreate
+      path: /etc/kubernetes/pki
+      type: Directory
   - name: kubernetes
     hostPath:
       path: /etc/kubernetes
       type: Directory
+  - name: pem-dir
+    hostPath:
+      path: /var/lib/kubelet/pki
+      type: Directory
   containers:
   - name: yurt-hub
     image: __yurthub_image__
     imagePullPolicy: IfNotPresent
     volumeMounts:
-    - name: hub-dir
-      mountPath: /var/lib/yurthub
     - name: kubernetes
       mountPath: /etc/kubernetes
+    - name: pki
+      mountPath: /etc/kubernetes/pki
+    - name: pem-dir
+      mountPath: /var/lib/kubelet/pki
     command:
     - yurthub
     - --v=2
     - --server-addr=__kubernetes_service_addr__
     - --node-name=$(NODE_NAME)
-    - --join-token=__join_token__
```

Comment:
Why delete parameter `--join-token` which generated when converted by yurtctl.

Reply:
`join-token` is not specified in `openyurt/config/yaml-templateyurthub.yaml`. I think it is not a must if we do not use hubself mode.

Reply:
@Windrow `join-token` can be generated by yurtctl, and we suggest that use hubself mode with `join-token` to generate yurthub client certificate that used to connect kube-apiserver.  so we need to keep `--join-token` parameter for yurthub.